### PR TITLE
fix(trigger): add size threshold to table bloat trigger

### PR DESCRIPTION
**Pull request by kqr**
Small tables that are frequently rewritten can trivially have > 50 % table bloat even though the absolute size of the bloat is just a few kB. This is the normal, stable, desirable condition for such tables. In those cases, the suggested VACUUM FULL is an incorrect response.

The common solution to this is to add a size threshold to the table bloat trigger. There's no strict guidance for such a threshold, and different sources use different thresholds:

- pgexperts/pgx_scripts needs 10 MB of table bloat to trigger by default
- keithf4/pg_bloat_check lists example triggers of 10 MB and 1 GB
- Heroku's documentation suggests a threshold of 100 MB

This commit is written with a conservative threshold to catch obvious false alarms without deviating too far from the function's earlier behaviour.

### Type of Change

- [ ] New health check
- [x] Bug fix
- [ ] Performance improvement
- [ ] Documentation update
- [ ] Refactoring/code cleanup
- [ ] Breaking change

## Related Issues

n/a

### PostgreSQL Version Compatibility

**Has this code been tested against the following PostgreSQL versions?**

- [ ] PostgreSQL 15
- [x] PostgreSQL 16
- [ ] PostgreSQL 17
- [ ] PostgreSQL 18

**Testing notes:**

Very small change and little scope for what could go wrong. It seems to work as expected. 

### Managed Database Platforms

**Has this code been deployed and tested on the following platforms?**

- [ ] Amazon RDS for PostgreSQL
- [ ] Google Cloud SQL for PostgreSQL (currently unable to test)
- [ ] Azure Database for PostgreSQL (currently unable to test)
- [ ] Neon 
- [ ] Supabase
- [ ] Self-managed PostgreSQL

**Platform-specific notes:**

Unfortunately not been able to do any testing on managed platforms.

## Additional Notes

<details><summary><h3>Confidence Score: 5/5</h3></summary>

This looks safe to merge.

No blocking issues found in the changed code. All three SQL variants apply the same threshold, and the predicate compares calculated bloat bytes against a valid PostgreSQL byte-size value.
</details>

<details><summary><h3><a href="https://www.greptile.com/trex">T-Rex</a> T-Rex Logs</h3></summary>

**What T-Rex did**
- Observed runtime blockers where connection variables were unset and key tooling like psql, postgres, initdb, pg\_ctl, createdb, pg\_prove, and Docker were unavailable.
- Compared paired logs that captured the same execution scope for HEAD^ and the current worktree to validate environment consistency.
- Executed the Python harness that enforces the size condition with bloat\_pct \> 50.0, preserves VACUUM FULL advice, tests boundary cases, and checks parity across all three installers.
- Validated that the harness results align with parity across installers and boundary-case expectations as demonstrated by the artifacts.
- Compiled and reviewed artifacts that document blocker evidence, paired logs, harness logic, and validation results.

<a href="https://app.greptile.com/trex/runs/15487459/artifacts"><picture><source media="(prefers-color-scheme: dark)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/ViewAllArtifactsDark.svg?v=4"><source media="(prefers-color-scheme: light)" srcset="https://greptile-static-assets.s3.amazonaws.com/badges/ViewAllArtifacts.svg?v=4">View all artifacts</picture></a>

<sub><a href="https://www.greptile.com/trex">T-Rex</a> Ran code and verified through T-Rex</sub>
</details>

<sub>Reviews (1): Last reviewed commit: ["fix(trigger): add size threshold to tabl..."](https://github.com/randoneering/pgfirstaid/commit/6a2bfc783edff68fe403a0840093c270875dc620) | [Re-trigger Greptile](https://app.greptile.com/api/retrigger?id=46459545)</sub>

### CODE REVIEW

File: view_pgFirstAid.sql [hunk 1]
```diff
@@ -545,7 +545,8 @@ from
 	q
 where
 	bloat_pct > 50.0
-and schemaname not like all(array['information_schema', 'pg_catalog', 'pg_toast', 'pg_temp%'])
```
Comment (kqr) [inline view_pgFirstAid.sql:548]: I adjusted the indentation of this filter so it's aligned with the rest of the where clause expression. Not strictly necessary, but I figured I might as well when I'm in there anyway.

**Comment by randoneering** [comment]
@greptileai

**Comment by randoneering** [comment]
@kqr thank you for the contribution. Edits look good, just don't mind me trying to fix CI 😎. I'll get it resolved and will approve both PRs and merge. Thank you for helping out!

**Review by randoneering — APPROVED** [review]
Looks good-thank you for the added measure in the bloat check  and formatting!
